test/EurobusinessTests.cpp

```diff
@@ -1,5 +1,6 @@
 #include <gtest/gtest.h>
-#include "Eurobusiness.hpp"
+#include "../src/Eurobusiness.hpp"
```

Comment:
Wystarczy samo #include "Eurobusiness.hpp", bo w CMake'u mamy include_directories(src)
Podobnie dla json.hpp poniżej.

Reply:
Poprawiłem to.